jupyter/minimal/ubi8-python-3.8/requirements.in

```diff
@@ -1 +1,60 @@
+# Minimal notebook requirements
 jupyterlab==3.2.4
+jupyter_server==1.4.1
+notebook==6.4.12
+setuptools==65.4.0
+pip==22.2.2
+
+# JupyterLab extensions
+# A JupyterLab extension for version control using git
+jupyterlab-git==0.39.2
+# Jupyter Kernel Gateway is a web server that supports different mechanisms for spawning and communicating with Jupyter kernels.
+jupyter_kernel_gateway==2.4.0
+# JupyterLab extension for dependency management and optimization created to guarantee reproducibility of Jupyter notebooks.
+jupyterlab-requirements>=0.6.4
+# This extension provides control over the notebook dependencies.The main goals of the project are the following: 1.manage notebook requirements without leaving the notebook 2.provide a unique and optimized* environment for each notebook
+jupyter-nbrequirements==0.7.3
```

Comment:
This is an extension for the classic notebook environment (`jupyter notebook`), which is deprecated in favor of `jupyter lab`. We don't need to install this extension.